wooey/templates/wooey/scripts/script_view.html

```diff
@@ -263,7 +263,10 @@ <h3 id="wooey-script-title">
           selectWooeyParser($('div[id^=parsergroup-]'))
         }
 
-        var $initial_parser = $("a[data-parser-pk=" + $wooey_parser.val() + "]");
+        // Set initial subparser if present in url params.
+        const urlParams = new URLSearchParams(window.location.search);
+        const subparser = urlParams.get('__subparser');
```

Comment:
is there a reason the param has __?

Reply:
It's because it is somewhat of an internal keyword that specifically controls the subparser to use, not to be confused with other URL parameters that are treated as positional or keyword arguments.